src/platforms/alexa/AlexaEvent.ts

```diff
@@ -46,25 +52,10 @@ export class AlexaEvent extends IVoxaEvent {
   };
 
   public requestToIntent: any = {
-    "AlexaSkillEvent.SkillDisabled": "AlexaSkillEvent.SkillDisabled",
```

Comment:
all this requests were being interpretted as intents... that's not correct, we don't need to execute the whole state machine for this